src/FacetedTokenInput.js

```diff
@@ -209,7 +218,9 @@ export default class FacetedTokenInput extends Component {
         this.refs.input.focus();
       }
       else if (tokenSelectionDirection === DIRECTION_NONE) {
-        this.refs['token' + tokenSelectionStart].focus();
+        if (this.refs['token' + tokenSelectionStart]) {
+          this.refs['token' + tokenSelectionStart].focus();
+        }
```

Comment:
This will correct the console error that appear when pressing backspace on an empty input.